apps/yara/CMakeLists.txt

```diff
@@ -14,22 +14,14 @@ message (STATUS "Configuring apps/yara")
 # Dependencies
 # ----------------------------------------------------------------------------
 
-# Disable older MSVC.
-if (MSVC AND MSVC_VERSION LESS 1800)
-    message (STATUS "  Visual Studio 2012 or newer required to build yara.")
-    return ()
-endif ()
-
-# Disable parallelism in MSVC as it supports only OpenMP 2.0.
-if (NOT MSVC)
```

Comment:
There's nothing to feel (un)responsible and whine. I only suggested to you all (not just to you you) to support Windows/ICC in order to distribute good Windows binaries. This seems to me more urgent than supporting platforms like Windows/c2 or OpenBSD/whatever. Also, please note that I am not suggesting to remove support for Windows/MSVC in favor of Windows/ICC. Thanks.

> On Mar 15, 2016, at 11:05 AM, Hannes Hauswedell <notifications@github.com> wrote:
> 
> In apps/yara/CMakeLists.txt <https://github.com/seqan/seqan/pull/1560#discussion_r56178216>:
> 
> > @@ -14,22 +14,14 @@ message (STATUS "Configuring apps/yara")
> >  # Dependencies
> >  # ----------------------------------------------------------------------------
> >  
> > -# Disable older MSVC.
> > -if (MSVC AND MSVC_VERSION LESS 1800)
> > -    message (STATUS "  Visual Studio 2012 or newer required to build yara.")
> > -    return ()
> > -endif ()
> > -
> > -# Disable parallelism in MSVC as it supports only OpenMP 2.0.
> > -if (NOT MSVC)
> Okay seriously, I don't like this solution either, so you can disable MSVC for now.
> 
> Will do.
> 
> Where and how do you do find_package now? Are OpenMP flags added to all targets?
> 
> Its in this diff, so feel free to do some easter egg searching ;-) Actually for BUILD_SYSTEM==DEVELOP we detect the dependencies once in build_system_init() and then add them to every target. For release builds everything is still done locally.
> 
> Why don't you make Windows/ICC builds a priority? Windows/MSVC builds suck because of OpenMP 2.0...
> 
> Let me think... I think it's because I am neither responsible for Intel stuff nor for Windows stuff  (although I have done 90% of making the library work with ICC).
> 
> I think @rrahn <https://github.com/rrahn>, @marehr <https://github.com/marehr> and @cpockrandt <https://github.com/cpockrandt> would all be more suited for the task. @cpockrandt <https://github.com/cpockrandt> and I tried to make "Clang with Microsoft Codegen"/C2 work to improve the situation on Windows, but cmake on Windows doesn't work with it, yet. Also we haven't even been given access to the Windows version of ICC, just Linux right now (I will discuss that with Intel folks here at IXPUG).
> 
> But I agree, we really should get an alternative to MSVC!
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/seqan/seqan/pull/1560/files#r56178216 <https://github.com/seqan/seqan/pull/1560/files#r56178216>